test/suppose-process.test.js

```diff
@@ -143,7 +118,7 @@ describe('process', function()
     })
   })
 
-  it('should call a method as response', function(done)
+  it_posixOnly('should call a method as response', function(done)
```

Comment:
No platform specific tests, if it doesn't work on a specific platform, then library needs to be fixed.